br/pkg/version/version.go

```diff
@@ -387,16 +387,14 @@ func ParseServerInfo(src string) ServerInfo {
 	var err error
 	serverInfo.ServerVersion, err = semver.NewVersion(versionStr)
 	if err != nil {
-		log.L().Warn("fail to parse version",
+		log.L().Warn("fail to parse version, fallback to 5.7.23",
 			zap.String("version", versionStr))
+		serverInfo.ServerVersion = semver.New("0.0.0")
```

Comment:
Why the following version is "0.0.0"?

Reply:
I've used `5.7.35` or so. I have no special preference. Do you have any good idea?

Reply:
Just feel a bit confused because we print a log which says we fallback to `5.7.23` above

Reply:
My mistake, resolved.